packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/index.stories.tsx

```diff
@@ -33,7 +32,7 @@ const metadata: ComponentMeta<typeof SwatchPicker> = {
   parameters: {
     docs: {
       description: {
-        component: [descriptionMd, bestPracticesMd].join('\n'),
```

Comment:
probably not best idea for sake of consistency of our index stories. cant we only remove the contents from the file instead completely removing ?

Reply:
I only removed bestPractices because it's empty. Currently all best practices should be on Fluent website 2

Reply:
do we have outdated storybook setup template then ?